client/www/scripts/application.js

```diff
@@ -2,12 +2,33 @@ angular.module('SteroidsApplication', [
     'supersonic',
     'firebase'
 ])
+
 .controller('IndexController', function($scope, supersonic) {
     $scope.navbarTitle = "Home";
+
+    $scope.turn_off_silence_mode = function() {
+        supersonic.logger.log("turn_off_silence_mode() called");
+
+        currentTime = new Date();
+        expireTime = currentTime;
+        expireTime.setMinutes(currentTime.getMinutes() -1);
+
+        var piRef = new Firebase('https://bellacoola.firebaseio.com/pi/');
```

Comment:
good JS practice is to declare all variables in the beginning of the function when the function is short